hasura/metadata/databases/ibcjuno/tables/public_token_unit.yaml

```diff
@@ -0,0 +1,43 @@
+table:
+  name: token_unit
+  schema: public
+object_relationships:
+- name: token
+  using:
+    foreign_key_constraint_on: token_name
+- name: token_price
+  using:
+    manual_configuration:
+      column_mapping:
+        denom: unit_name
+      insertion_order: null
+      remote_table:
+        name: token_price
+        schema: public
+- name: token_ibc_denom
```

Comment:
Are we sure this object relationship should be here? Don't we have an array relationship for this?